rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -705,17 +702,27 @@ void configure_qos_for_security(
     qos, "dds.sec.access.permissions", "permissions.p7s",
     security_options);
 
-  dds_qset_prop(qos, "dds.sec.auth.library.path", "libdds_security_auth.so");
+  dds_qset_prop(qos, "dds.sec.auth.library.path", "dds_security_auth");
   dds_qset_prop(qos, "dds.sec.auth.library.init", "init_authentication");
   dds_qset_prop(qos, "dds.sec.auth.library.finalize", "finalize_authentication");
 
-  dds_qset_prop(qos, "dds.sec.crypto.library.path", "libdds_security_crypto.so");
+  dds_qset_prop(qos, "dds.sec.crypto.library.path", "dds_security_crypto");
   dds_qset_prop(qos, "dds.sec.crypto.library.init", "init_crypto");
   dds_qset_prop(qos, "dds.sec.crypto.library.finalize", "finalize_crypto");
 
-  dds_qset_prop(qos, "dds.sec.access.library.path", "libdds_security_ac.so");
+  dds_qset_prop(qos, "dds.sec.access.library.path", "dds_security_ac");
   dds_qset_prop(qos, "dds.sec.access.library.init", "init_access_control");
   dds_qset_prop(qos, "dds.sec.access.library.finalize", "finalize_access_control");
+
+  return RMW_RET_OK;
+#else
+  (void) qos;
+  (void) security_options;
+  RMW_SET_ERROR_MSG(
+    "Security was requested but this Cyclone version does not have security "
+    "support enabled. Recompile with CMake option '-DENABLE_SECURITY'");
```

Comment:
Small suggestion to rephrase slightly and make it clear that the issue is resolved with a Cyclone rebuild, not an RMW rebuild.

```suggestion
    "Security was requested but the Cyclone DDS being used does not have security "
    "support enabled. Recompile Cyclone DDS with the '-DENABLE_SECURITY' CMake option");
```

Reply:
Updated.